setup.cfg

```diff
@@ -41,9 +41,9 @@ dev =
     kivy_deps.gstreamer_dev~=0.3.3; sys_platform == "win32"
     kivy_deps.sdl2_dev~=0.8.0; sys_platform == "win32"
     kivy_deps.glew_dev~=0.3.1; sys_platform == "win32"
-    flake8
     pre-commit
     responses
+    ruff
```

Comment:
How about pinning a version (can be relaxed in both files, and not necessarily `~=`) as `cicd-requirements.txt` ?

Reply:
Does this mean `ruff==0.8.3` (done now) or `ruff==0.8.x` or something else.

Reply:
`~=` allows `0.8.*`, so is more relaxed than `==`.

I would suggest a `> 0.8.3` and `< 1.0.0` here, as a breakage does not impact any user, but it might just impact CI/CD pipelines and dev tests.